app/components/index.tsx

```diff
@@ -1,17 +1,10 @@
 import React from 'react'
-import { StyleSheet, Text as DefaultText, TextInput as DefaultTextInput, View as DefaultView } from 'react-native'
+import { TextInput as DefaultTextInput, View as DefaultView } from 'react-native'
+import { tailwind } from '../tailwind'
+import { Text as DefaultText, TextProps } from './Text'
 
-const Default = StyleSheet.create({
-  text: {
-    fontSize: 16,
-    fontWeight: '600'
-  }
-})
-
-export function Text (props: DefaultText['props']): JSX.Element {
-  const { style, ...otherProps } = props
-
-  return <DefaultText style={[Default.text, style]} {...otherProps} />
+export function Text (props: TextProps): JSX.Element {
+  return <DefaultText {...props} />
 }
```

Comment:
could just export as this for conveience

```ts
export * from './Text'
```

Reply:
Updated